### PR TITLE
# 2653 Musical Floor Game

### LINKED ISSUE #2653: [New game]: Musical floor game
### 🎮 Game Request

The user has to hover over the colourful tiles to produce music.

### Point down the features

The user has to hover over the colourful tiles to produce music.

### Select program in which you are contributing

GSSoC23

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by PVBharadwaj**
## PR Description 📜

I added the musical floor game where users hover over the colourful tiles to produce sound.
Fixes #2653 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @PVBharadwaj ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by PVBharadwaj** [comment]
@kunjgit  please review this pr under gssoc'23

**Comment by PVBharadwaj** [comment]
@kunjgit please review this pr under gssoc'23

**Comment by PVBharadwaj** [comment]
@kunjgit  please review this pr under gssoc'23. It has been 3 days since I raised the PR.

File: Games/Musical_Floor/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dancing Girl Animation</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="dance-girl"></div>
+</body>
+</html>
```
Comment (gurjeetsinghvirdee) [inline Games/Musical_Floor/index.html:12]: Remove this extra code

Comment (PVBharadwaj) [reply Games/Musical_Floor/index.html]: Removed

**Comment by PVBharadwaj** [comment]
@gurjeetsinghvirdee  please review this PR under GSSoC'23. I have made the required changes.

**Review by gurjeetsinghvirdee — APPROVED** [review]

**Comment by kunjgit** [comment]
Thank you  @PVBharadwaj , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**